docs/docs/guides/how-to-compose-your-own-testnet.md

```diff
@@ -0,0 +1,215 @@
+
+**What you will do:**
+
+1. Create a local Kurtosis package template
+2. Within the template, define how your private Ethereum testnet should look like. This example will leverage the Lighthouse CL client and Geth EL client to build a single, full staking node over Docker with [`eth-kurtosis`](https://github.com/kurtosis-tech/eth-kurtosis/tree/main).
+3. Launch the private testnet locally over Docker
+4. Push everything to Github as a new repository for others to use
+
+**What you will need beforehand to get started:**
+
+- Install [Kurtosis](https://docs.kurtosis.com/install) (or [upgrade to latest](https://docs.kurtosis.com/upgrade) if you already have it)
+- Install [Docker & ensure its running](https://docs.kurtosis.com/install#i-install--start-docker)
+- A Github account to leverage the [template repository](https://github.com/kurtosis-tech/package-template-repo)
+
+### 1. Set up a empty Kurtosis package
+To begin, create and `cd` into a directory to hold your files:
+```bash
+mkdir my-testnet && cd my-testnet 
+```
+Next, create a file called `network_params.json` in that folder with the following contents:
+```json
+{
+  "preregistered_validator_keys_mnemonic": "giant issue aisle success illegal bike spike question tent bar rely arctic volcano long crawl hungry vocal artwork sniff fantasy very lucky have athlete",
+  "num_validator_keys_per_node": 64,
+  "network_id": "3151908",
+  "deposit_contract_address": "0x4242424242424242424242424242424242424242",
+  "seconds_per_slot": 12,
+  "genesis_delay": 10
+}
+```
+The contents above will be used to define the specific parameters with which to start the network with. 
+
+Finally, create a `kurtosis.yml` file in the same folder with the following contents, replacing `$YOUR_GITHUB_USERNAME` with your actual Github username.
+```yml
+name: github.com/$YOUR_GITHUB_USERNAME/my-testnet
+```
+
+Awesome. You have just created your very own [Kurtosis package](https://docs.kurtosis.com/concepts-reference/packages/)! This package will form the backbone of the environment definition you will use to instantiate and deploy your private testnet. A Kurtosis package is completely reproducible, modular, and will work over Docker locally or in the cloud on Kubernetes. 
+
+### 2. Import dependencies
+Now that you have a local project to house your definition and some parameters to start the network with, its time to actually build the network. First, create a Starlark file called `main.star` and add the following three lines:
+```python
+// main.star
```

Comment:
lets make this `# main.star` so that its a valid python comment. currnetly copying and pasting this is invalid